src/lib/sql/sql_pipeline_statement.cpp

```diff
@@ -217,4 +225,47 @@ const std::shared_ptr<TransactionContext>& SQLPipelineStatement::transaction_con
 }
 
 const std::shared_ptr<SQLPipelineStatementMetrics>& SQLPipelineStatement::metrics() const { return _metrics; }
+
+void SQLPipelineStatement::_verify_lqp() {
+  if (!_unoptimized_logical_plan) {
```

Comment:
a "verification" function should never issue significant work like SQL parsing/translation is. Calling `get_unoptimized_logical_plan();` here should not be necessary if _verify_lqp() was called from `get_unoptimized_logical_plan()` itself